common/app/conf/switches/PerformanceSwitches.scala

```diff
@@ -148,8 +148,8 @@ trait PerformanceSwitches {
 
   val DiscussionSwitch = Switch(
     SwitchGroup.Performance,
-    "discussion",
-    "If this switch is on, comments are displayed on articles. Turn this off if the Discussion API is blowing up.",
+    "discussion-visible-web",
+    "If this switch is on, comments are displayed on articles on web. This switch only removes the UI from the articles on web and does not affect requests to DAPI.",
```

Comment:
It also doesn't affect the endpoint on frontend. It only hide/show the UI but endpoints are still reachable. The description should emphasize it only act on the UI, the underlying feature are still available